src/app/app.component.ts

```diff
@@ -9,14 +9,15 @@ import { LoginPage } from '../pages/login/login';
 import { SignupPage} from '../pages/signup/signup';
 import { ItemPage } from '../pages/item/item';
 import { FirebaseTestPage } from '../pages/firebase-test/firebase-test';
+import {LogindetailPage} from '../pages/logindetail/logindetail';
```

Comment:
For consistency, spaces between curly brackets would be good i.e. { Module } instead of {Module}